spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java

```diff
@@ -265,7 +272,8 @@ public MultiListenerBean multiListener() {
 
 		@Bean
 		public ProducerFactory<Integer, String> producerFactory() {
-			return new DefaultKafkaProducerFactory<>(producerConfigs());
+			Map<String, Object> configs = producerConfigs();
+			return new DefaultKafkaProducerFactory<>(configs);
```

Comment:
Technically I don't see difference with existing code and your change.
Just some unnecessary noise...
Will revert on merge.